### PR TITLE
Add option to specify MaxAuthTries in sshd_configuration

### PR DESCRIPTION
**Pull request by jthiesfeld**
This is my first attempt to contribute on github or to public puppet modules.  Please let me know if I need to make any additional changes.

### CODE REVIEW

**Comment by jthiesfeld** [comment]
Any hints as to how I might resolve these failures?

Evaluation Error: Unknown variable: '::operatingsystem'. at /home/travis/build/ghoneycutt/puppet-module-ssh/spec/fixtures/modules/ssh/manifests/init.pp:146:15 on node testing-worker-linux-docker-59cf05a1-3380-linux-11.prod.travis-ci.org

It looks like only the strict tests are failing.

**Comment by ghoneycutt** [comment]
Try adding architecture and perhaps operatingsystem to the default facts at the very top.

**Comment by jthiesfeld** [comment]
Are you suggesting putting the following somewhere between lines 126 and 143 of manifests/init.pp?  This appears to be unrelated to my changes, so I don't understand why it is a problem introduced with my build.

$::operatingsystem == 'SLES'

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-ssh",
-  "version": "3.37.0",
+  "version": "3.38.0",
```
Comment (ghoneycutt) [inline metadata.json]: don't bump versions in your PR's. I might merge someone else's first, cut a release, then merge this and cut a release and then the version here would be wrong.

**Comment by ghoneycutt** [comment]
Thanks for all the work! I cleaned it up in PR #170

**Pull request closed without merging**